.github/workflows/e2e-tests.yml

```diff
@@ -8,32 +8,20 @@ on:
 
 jobs:
 
-  docker-build-e2e:
-    runs-on: ubuntu-20.04
-    steps:
-      - name: Delete unnecessary tools folder to prevent "out of space" error
-        run: rm -rf /opt/hostedtoolcache
```

Comment:
this step is deprecated? if yes, why? I see that you didn't put it in the new workflow

Reply:
Good catch, I will move this step to the docker-build-e2e workflow.